star-rx72n-firmware/lib/rx_hcsr04/src/rx_hcsr04_hal_mock.c

```
@@ -57,6 +190,34 @@ rx_err_t hcsr04_hal_gpio_deinit(gpio_pin_t pin)
 
 void hcsr04_hal_delay_us(uint32_t us)
```

Clang (14.0.6)

[warning] parameter name 'us' is too short, expected at least 3 characters

(readability-identifier-length)